pontoon/base/models.py

```diff
@@ -951,6 +961,38 @@ def serialize(self):
         }
 
 
+class TranslationMemoryEntryManager(models.Manager):
+    def minimum_levenshtein_ratio(self, text, min_quality=0.7):
+        """
+        Returns entries that match minimal levenshtein_ratio
+        """
+        length = len(text)
+        min_dist = math.ceil(max(length * min_quality, 2))
+        max_dist = math.floor(min(length / min_quality, 1000))
+
+        # Only check entities with similar length
+        entries = self.extra(
+            where=['(CHAR_LENGTH(source) BETWEEN %s AND %s)'
+                  ' AND (char_length(source) + char_length(%s)-levenshtein(source, %s, 1, 2, 2))::float/(char_length(source) + char_length(%s)) > %s'],
+            params=(min_dist, max_dist, text, text, text, min_quality),
+            select={'quality': '(char_length(source) + char_length(%s)-levenshtein(source, %s, 1, 2, 2))::float/(char_length(source) + char_length(%s)) * 100'},
```

Comment:
Unfortunately, it looks like that calculated columns aren't available for filtering.